Gemfile

```diff
@@ -29,6 +29,7 @@ gem 'shoryuken', require: nil
 gem 'sidekiq', require: nil
 gem 'simplecov', require: false, group: :test
 gem 'simplecov-cobertura', require: false, group: :test
+gem "sneakers", "~> 2.12.0", require: nil
```

Comment:
We could also make this `2.12` so that we'll just test against the latest in the 2.x series. Do you see a reason why we should only test `2.12.x`?